lib/check_graphite/version.rb

```diff
@@ -1,3 +1,3 @@
 module CheckGraphite
-  VERSION = "0.2.5"
+  VERSION = "0.3.0.pre1"
```

Comment:
While I agree on the principle, I hold that it is an absolute rule that you must never publish software without documenting the fact and version that you have done so. Since this branch is ultimately destined for upstream repo, I did not want to merge it to burcorp master, and so the version is bumped here and tagged accordingly. This is acceptable for release candidates but not for finals.